backend/sirius-web-tests/src/test/java/org/eclipse/sirius/web/tests/GeneralPurposeTestCases.java

```diff
@@ -75,19 +77,19 @@
     private static final String HEIGHT_100 = "height: 100%;"; //$NON-NLS-1$
 
     //// @formatter:off
-    private static final List<String> COPYRIGHT_HEADER = List.of(
-            "/*******************************************************************************", //$NON-NLS-1$
-            " * Copyright (c) 2019, 2020 Obeo.", //$NON-NLS-1$
-            " * This program and the accompanying materials", //$NON-NLS-1$
-            " * are made available under the terms of the Eclipse Public License v2.0", //$NON-NLS-1$
-            " * which accompanies this distribution, and is available at", //$NON-NLS-1$
-            " * https://www.eclipse.org/legal/epl-2.0/", //$NON-NLS-1$
-            " *", //$NON-NLS-1$
-            " * SPDX-License-Identifier: EPL-2.0", //$NON-NLS-1$
-            " *", //$NON-NLS-1$
-            " * Contributors:", //$NON-NLS-1$
-            " *     Obeo - initial API and implementation", //$NON-NLS-1$
-            " *******************************************************************************/"); //$NON-NLS-1$
+    private static final List<Pattern> COPYRIGHT_HEADER = List.of(
+            Pattern.compile(Pattern.quote("/*******************************************************************************")), //$NON-NLS-1$
+            Pattern.compile(" \\* Copyright \\(c\\) [0-9]{4}(, [0-9]{4})* Obeo\\."), //$NON-NLS-1$
+            Pattern.compile(Pattern.quote(" * This program and the accompanying materials")), //$NON-NLS-1$
+            Pattern.compile(Pattern.quote(" * are made available under the terms of the Eclipse Public License v2.0")), //$NON-NLS-1$
+            Pattern.compile(Pattern.quote(" * which accompanies this distribution, and is available at")), //$NON-NLS-1$
+            Pattern.compile(Pattern.quote(" * https://www.eclipse.org/legal/epl-2.0/")), //$NON-NLS-1$
+            Pattern.compile(Pattern.quote(" *")), //$NON-NLS-1$
+            Pattern.compile(Pattern.quote(" * SPDX-License-Identifier: EPL-2.0")), //$NON-NLS-1$
+            Pattern.compile(Pattern.quote(" *")), //$NON-NLS-1$
+            Pattern.compile(Pattern.quote(" * Contributors:")), //$NON-NLS-1$
+            Pattern.compile(Pattern.quote(" *     Obeo - initial API and implementation")), //$NON-NLS-1$
+            Pattern.compile(Pattern.quote(" *******************************************************************************/"))); //$NON-NLS-1$
```

Comment:
Don't you have a separate PR for that?

Reply:
Yes, but I needed it here to get the build not to fail too early and see if there were other failing tests. If the other PR is merged, I'll rebase this one and this particular commit will disappear from here.